site/content/installation/ngf-images/jwt-token-docker-secret.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Getting the NGINX Plus Gateway Fabric Image with JWT
+description: "Use your NGINX Gateway Fabric subscription JWT token to get the NGINX Plus Gateway Fabric image from the F5 Docker registry."
+weight: 150
+doctypes: ["install"]
+toc: true
+---
+
+{{<custom-styles>}}
+
+## Overview
+
+Follow the steps in this document to pull the NGINX Plus Gateway Fabric image from the F5 Docker registry into your Kubernetes cluster using your JWT token.
+
+{{<important>}}An NGINX Plus subscription certificate and key will not work with the F5 Docker registry. For NGINX Gateway Fabric, you must have an Connectivity Stack for Kubernetes subscription.
+
+To list the available image tags using the Docker registry API, you will also need to download the NGINX Plus Gateway Fabric (per instance) certificate (`nginx-repo.crt`) and the key (`nginx-repo.key`) from [MyF5](https://my.f5.com).{{</important>}}
+
+## Before You Begin
+
+You will need the following information from [MyF5](https://my.f5.com) for these steps:
+
+- A JWT Access Token (per instance) for NGINX Gateway Fabric from an active NGINX Gateway Fabric subscription.
+- The certificate (`nginx-repo.crt`) and key (`nginx-repo.key`) for each NGINX Gateway Fabric instance, used to list the available image tags from the Docker registry API.
+
+## Prepare NGINX Gateway Fabric
+
+1. Log into the [MyF5 Portal](https://my.f5.com/), navigate to your subscription details, and download the relevant .cert, .key and .JWT files.
+
+## Using the JWT token in a Docker Config Secret
+
+1. Create a Kubernetes `docker-registry` secret type on the cluster, using the JWT token as the username and `none` for password (as the password is not used).  The name of the docker server is `private-registry.nginx.com`.
```

Comment:
I would say that this is an example server, and should be replaced with the server for the user (unless we plan on updating this to the real server?)

Reply:
This is the real server, it's the F5 managed NGINX container registry

Reply:
Ok thank you, thought the name was so generic, it couldn't have been real! :)